README.md

```diff
@@ -89,6 +89,18 @@ let device: Device = instance
     .unwrap();
 ```
 
+To not loose this lifetime single items can be "cast" to a slice of length _one_ with `std::slice::from_ref` while still taking advantage of `Deref`:
```

Comment:
Interesting, I didn't actually knew that you could do that.

Btw it is "lose"

Reply:
Ah, great catch, sorry!